src/Controller/AdminExtensionControllerTrait.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace AlterPHP\EasyAdminExtensionBundle\Controller;
+
+use AlterPHP\EasyAdminExtensionBundle\Security\AdminAuthorizationChecker;
+use EasyCorp\Bundle\EasyAdminBundle\Event\EasyAdminEvents;
+use League\Uri\Modifiers\RemoveQueryParams;
+use League\Uri\Schemes\Http;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+trait AdminExtensionControllerTrait
+{
+    public static function getSubscribedServices(): array
```

Comment:
This part should be kept in the concrete controller IMO. This will conflict anyway as you will get it in both traits.

Please move static getSubscribedServices to the controller

Reply:
Yes it makes sense I pushed a new commit, though the other Extension do not use getSubscribedServices, it can be useful with another that use it.